### ISSUE TITLE
DUP: #776 - Warp not responding to windows manager

### ISSUE THREAD

**Issue by ab-smith** [warpdotdev/Warp#806]
### Describe the bug

```shell
Hello,

Since two or three updates ago, I cannot manage warp window with any of the tools on Mac (Magnet or equivalent).
I thought it was a key binding issue but even the actions on drop menu (from the status icon) are disabled.

Thank you
```

### To Reproduce

Steps to reproduce:
1. have a windows manager installed on Mac (Magnet for instance)
2. use the keys shortcut (like control+option+enter or control+option+left to full screen or left align)
3. no effect was observed
4. try the magnet icon from the tray/status bar and notice that actions for warp are disabled.

### Screenshots

<img width="1242" alt="image" src="https://user-images.githubusercontent.com/11318384/157837895-16714ee7-3f88-4547-982f-7cb73400ccae.png">

### Operating System

MacOS

### OS Version

12.2.1

### Additional context

```shell
Apple silicon mac book pro
```

**Comment by dominickm** [comment]
I am having this issue as well

**Comment by dbrockman** [comment]
+1, I'm on an Intel MacBook Pro with macOS 12.2.1 and I'm also using Magnet to move and resize windows.

**Comment by remidej** [comment]
It happens to me too, I'm on version `v0.2022.03.07.08.51.stable_01` on macOS. I'm not sure which release introduced the issue but I'm pretty sure didn't have that problem before.

For more context, there's a section about this on [Magnet's FAQ](https://magnet.crowdcafe.com/faq.html):

>**Why is my window not resizing?**
>
>Magnet can handle all standard resizable windows in OS X for you. Magnet can not do anything with non-standard windows used by some developers. Fortunately, such windows are quite rare. Adobe Creative Suite & Creative Cloud apps older than 2017 version is not supported exactly for this reason.

**Comment by jdh313** [comment]
Can confirm I'm having the same issue with Swish. For some reason it still works with Magnet, though.

That being said, this is a deal breaker for me until a fix is available.

**Comment by gnapse** [comment]
I have the same issue. I encourage everyone having this issue to upvote it with a 👍 in the initial issue comment, as this is encouraged by the Warp feedback form

<img width="755" alt="image" src="https://user-images.githubusercontent.com/15199/159125132-6e595f13-5105-460b-aa5a-3197e2b550b3.png">

**Comment by gnapse** [comment]
BTW, this is a duplicate of #776

**Comment by ab-smith** [comment]
> BTW, this is a duplicate of #776

sorry apparently missed it when I was looking for the existing one and looks like the title has been changed as well

**Comment by ab-smith** [comment]
> Can confirm I'm having the same issue with Swish. For some reason it still works with Magnet, though.
> 
> That being said, this is a deal breaker for me until a fix is available.

what is your setup please (magnet and warp versions) as magnet is not working on my side?

**Comment by Ghirigoro** [comment]
I don't use Magnet (I use Hammerspoon for window management) but I've also been experiencing this. In my case at least it seems to be because Warp terminals have been changed to not be "standard" windows. All other application windows I've checked are defined as `AXWindow`s whereas Warp windows are `TextArea`s (actually it looks like the Warp app windows are `TextArea`s which contain each block as a series of `Text` internal windows but I could be wrong about this last bit). Hopefully that's something that can be changed back.

A workaround (at least for Hammerspoon) is to not filter by standard windows (i.e. allow the window manager to interact with all windows including non-standard ones like dialogs).
